### PR TITLE
Update US.m3u (480p Add)

### PR DESCRIPTION
**Pull request by Central302**
This Patch Will Add 480p Version of  Some USA OTA Station Stream from Canada Server.
Aspectratio Will 4:3.

(Also, Removed Some Link Duplicate.)

### CODE REVIEW

**Comment by Dum4G** [comment]
Why do we have some streams being removed as well? Those look fine to me

File: channels/us.m3u [hunk 1]
```diff
@@ -1235,26 +1233,14 @@ http://encodercdn1.frontline.ca/encoder184/output/NBC_Spokane_720p/playlist.m3u8
...
-#EXTINF:-1 tvg-id="WNBCNews.us" tvg-country="US" tvg-language="English" tvg-logo="https://github.com/Tapiosinn/tv-logos/raw/master/countries/united-states/us-local/nbc-4-wnbc-us.png" group-title="Local",NBC4 WNBC-TV New York News (1080p) [Offline]
-https://wnbclive-f.akamaihd.net/i/NY_STREAM1@13992/master.m3u8
 #EXTINF:-1 tvg-id="WRCNews.us" tvg-country="US" tvg-language="English" tvg-logo="" group-title="Local",NBC4 WRC-TV Washington News (1080p)
 https://wrclive-f.akamaihd.net/i/DC_STREAM1@22925/master.m3u8
 #EXTINF:-1 tvg-id="KXASNews.us" tvg-country="US" tvg-language="English" tvg-logo="" group-title="Local",NBC5 KXAS-TV News Dallas TX (1080p) [Not 24/7]
 https://kxaslive-f.akamaihd.net/i/DALLAS_STREAM5@5495/master.m3u8
-#EXTINF:-1 tvg-id="KXASNews.us" tvg-country="US" tvg-language="English" tvg-logo="" group-title="Local",NBC5 KXAS-TV News Dallas TX (1080p) [Not 24/7]
-https://kxaslive-f.akamaihd.net/i/DALLAS_STREAM5@5495/master.m3u8
-#EXTINF:-1 tvg-id="WTVJNews.us" tvg-country="US" tvg-language="English" tvg-logo="" group-title="Local",NBC6 WTVJ-TV News Miami FL (1080p) [Not 24/7]
-https://wtvjlive-f.akamaihd.net/i/MIAMI_STREAM1@19309/master.m3u8
```
Comment (Dum4G) [inline channels/us.m3u:1257]: Seems to be online

File: channels/us.m3u [hunk 2]
```diff
@@ -120,8 +120,6 @@ https://content.uplynk.com/channel/ext/72750b711f704e4a94b5cfe6dc99f5e1/wabc_24x
 http://encodercdn1.frontline.ca/yavin/output/ABC_Seattle_720p/playlist.m3u8
 #EXTINF:-1 tvg-id="KXLYTV.us" tvg-country="US" tvg-language="English" tvg-logo="https://raw.githubusercontent.com/Tapiosinn/tv-logos/master/countries/united-states/abc-logo-2013-butterscotch-us.png" group-title="Local",ABC Spokane WA (KXLY-TV1) (720p)
 http://encodercdn1.frontline.ca/kamino/output/ABC_Spokane_720p/playlist.m3u8
-#EXTINF:-1 tvg-id="KXLY-TV.us" tvg-country="US" tvg-language="English" tvg-logo="https://raw.githubusercontent.com/Tapiosinn/tv-logos/master/countries/united-states/abc-logo-2013-butterscotch-us.png" group-title="Local",ABC Spokane WA (KXLY-TV1) (720p)
-http://encodercdn1.frontline.ca/kamino/output/ABC_Spokane_720p/playlist.m3u8
```
Comment (Dum4G) [inline channels/us.m3u:124]: Took a minute to load but it's live

**Review by Dum4G — CHANGES_REQUESTED** [review]

File: channels/us.m3u (hunk 2 above)
Comment (Central302) [reply channels/us.m3u:124]: Sorry, Removed for Duplicate.

File: channels/us.m3u (hunk 1 above)
Comment (Central302) [reply channels/us.m3u:1257]: Removed for Duplicate.

**Review by BellezaEmporium — APPROVED** [review]

**Review by Dum4G — APPROVED** [review]

**Pull request merged**